src/node-lib/curriculum-api-2023/queries/pupilLessonOverview/pupilLessonOverview.query.ts

```diff
@@ -29,7 +31,22 @@ export const pupilLessonOverviewQuery =
       });
     }
 
+    const isLegacyLicense =
+      !lesson.lessonCohort || lesson.lessonCohort === LEGACY_COHORT;
+    let transcriptSentences: string | string[] =
+      lesson.transcriptSentences ?? [];
+
+    if (lesson.videoTitle && !isLegacyLicense) {
+      // For new content we need to fetch the captions file from gCloud and parse the result to generate
+      // the transcript sentences.
+      const fileName = `${lesson.videoTitle}.vtt`;
+
+      transcriptSentences = (await getCaptionsFromFile(fileName)) ?? [];
+    }
+
     return pupilLessonOverviewSchema.parse({
       ...lesson,
+      transcriptSentences,
+      isLegacyLicense,
```

Comment:
I believe we can use this to decide which licence to display on the lesson intro page as well as to decide if this is an new  video (and that we need to get the captions from a bucket).